bpf/lib/nodeport.h

```diff
@@ -1217,13 +1222,17 @@ static __always_inline int nodeport_nat_ipv4_fwd(struct __ctx_buff *ctx)
 		.addr = 0,
 	};
 	int ret = CTX_ACT_OK;
+	__u32 monitor = ctx_load_meta(ctx, CB_CT_MONITOR);
 
 	if (snat_v4_needed(ctx, &target.addr, &from_endpoint))
 		ret = snat_v4_process(ctx, NAT_DIR_EGRESS, &target,
 				      from_endpoint);
+
 	if (ret == NAT_PUNT_TO_STACK)
 		ret = CTX_ACT_OK;
 
+	send_trace_notify(ctx, TRACE_TO_NETWORK, 0, 0, 0, 0, ret, monitor);
```

Comment:
I think we should skip the commit that adds `TRACE_TO_NETWORK` traces for now. See the discussion at https://github.com/cilium/cilium/pull/14696. These additional traces will get very noisy quickly so there's a bit more work needed before we can introduce them.